lang/user/ms.xml

```diff
@@ -627,9 +618,11 @@ Pilih butang 'Enter' awak.</select_enter_button>
 Sesetengah aplikasi tidak menggunakan ini dan mendapatkan butang pengesahan lalai.</enter_button_description>
 			<circle>Bulatan</circle>
 			<cross>X</cross>
+			<pstv_mode>Mode PS TV</pstv_mode>
 			<pstv_mode_description>Tandai kotak untuk mendayakan mode PS TV Emulated.</pstv_mode_description>
 			<show_mode>Mode Show</show_mode>
 			<show_mode_description>Tandai kotak untuk mendayakan mode Show.</show_mode_description>
+			<demo_mode>Mode Demo</demo_mode>
```

Comment:
Delete both, no need add if keep english 👌

Reply:
But not the same, what do you think @Kitakatarashima .

Reply:
eh, it's a language, Indonesian. Let me explain that there are several words that are the same as English

Reply:
If the English language is updated, yes, in Indonesia it is still updated in Indonesian, which is nice to look at